packages/types/src/chain-adapters/index.ts

```diff
@@ -1,3 +1,4 @@
+import { btcChainId, cosmosChainId, ethChainId, osmosisChainId } from '@shapeshiftoss/caip'
```

Comment:
Another thought is while we are doing all of this type change/refactor, would this be a good time to move chain-adapters specific types back into the chain-adapter package and ensure only shared types live in the types package?